### PR TITLE
App view

### PR DESCRIPTION
**Pull request by Smart1-hub**
Implement the design from the sneak peek wireframes.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -15,9 +11,11 @@
   <section class="user_post">
     <% @user.posts.each_with_index do |post, idx|%>
       <div>
-        <h3>Post#<%=idx%>: <%= post.title %> </h3>
+        <h3>Post <%=idx+1%>: <%= post.title %></h3>
```
Comment (brytebee) [inline app/views/users/show.html.erb:14]: 👏👏👏👏
Great use of `Ruby` syntax to display a count/numbering of the users. Well done!

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
-
     <div class="post_details">
       <h3><%= link_to i.title, user_post_path(@user.id, i) %></h3>
       <p><%= i.text %></p>
     </div>
 
-    <div>
-      <%= i.comments.limit(3).each do |idx| %>
-        <p><%= idx.text %> By: <%= idx.user.name %></p>
+    <div class ="post_comment">
```
Comment (brytebee) [inline app/views/posts/index.html.erb:11]: **NB** The issue raised in the summary also applies here. `Post` (singular) would be better than `i`.

**Review by brytebee — APPROVED** [review]
Hi, @ 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
### I left a/some comment(s) here because the files weren't changed in the current milestone. 
- https://github.com/Smart1-hub/blog_app/blob/57024e702f40b384583a7c67e2f611cd655c060f/app/views/users/index.html.erb#L3-L8
    In future projects, kindly use descriptive block variables for the function they have in the application. This gives a more readable codebase not just to other developers but to the future you - when you return to the codebase.
    In this case, `user` (singular) would be a better option than `i`. Don't you think so❓ 

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (Smart1-hub) [reply app/views/posts/index.html.erb:11]: Thanks, observations noted

**Pull request merged**